pegasus/sites.v3/code.org/views/ie_unsupported.haml

```diff
@@ -0,0 +1,38 @@
+#warning-banner{style: 'display: none;'}
+  #message
+    %i.fa.fa-warning.warning-sign
+    &nbsp;
+    = I18n.t('compatibility.upgrade.ie_message')
+    &nbsp;
+    %a{href: 'https://support.code.org/hc/en-us/articles/219557867', target: '_blank'}
+      = I18n.t('compatibility.upgrade.link')
+  #hide-warning
+    %i.fa.fa-times.close
+
+:javascript
+  function setCookie(key, value, days) {
```

Comment:
pity we don't have a good way to share js like this (unless we hoist over to apps or code-studio, which seems odd)